packages/web3-eth-contract/test/integration/contract_methods_errors.test.ts

```diff
@@ -101,5 +122,27 @@ describe('contract errors', () => {
 				},
 			});
 		});
+
+		it('should catch error with parameter using PromiEvent.on("error")', async () => {
+			const expectedThrownError = {
```

Comment:
I think it would be nice if we have this `const expectedThrownError = {...` above and we use it inside the 2 related test cases which related to the `CustomError` error.